src/components/search/Search.tsx

```diff
@@ -0,0 +1,75 @@
+import classNames from 'classnames';
+import * as React from 'react';
+
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import './Search.scss';
+
+interface SearchTypes {
+  inMenu?: boolean;
+  className?: string;
+}
+
+const Search = (props: SearchTypes) => {
+
+  const { inMenu, className } = props;
```

Comment:
I think we should lose the upper newline, but keep a newline between the variables and return statement for better readbility.

```suggestion
  const { inMenu, className } = props;

```

Reply:
fixed, thank you.